pages/content/amp-dev/documentation/guides-and-tutorials/integrate/_blueprint.yaml

```diff
@@ -9,3 +9,4 @@ bucket: true
 formats:
   - websites
   - email
+  - stories
```

Comment:
Is it still needed?

Reply:
Actually the "embed stories" entry here accepts the format stories so I wonder why this list didn't have stories in it... @CrystalOnScript any clue? Should we add it if there's articles targeted to the stories format in this category?

Reply:
Great call @mszylkowski - let's merge it in with this change. Thanks for catching that!